### PR TITLE
Dockerfile: fix location to FreeBSD files

### PR DESCRIPTION
**Pull request by halkyon**

### CODE REVIEW

**Review by jenlij — APPROVED** [review]

File: Dockerfile [hunk 1]
```diff
@@ -59,8 +59,8 @@ RUN wget https://github.com/karalabe/xgo/blob/master/docker/base/patch.tar.xz?ra
 
 # Setup Freebsd, stolen from https://github.com/sandvine/freebsd-cross-build
 RUN mkdir -p /freebsd/x86_64-pc-freebsd12 && cd /freebsd \
- && wget ftp://ftp-archive.freebsd.org/pub/FreeBSD/releases/amd64/amd64/12.3-RELEASE/base.txz \
- && wget ftp://ftp-archive.freebsd.org/pub/FreeBSD/releases/amd64/amd64/12.3-RELEASE/lib32.txz \
+ && wget http://ftp-archive.freebsd.org/pub/FreeBSD-Archive/old-releases/amd64/12.3-RELEASE/base.txz \
```
Comment (egonelbre) [inline Dockerfile:62]: Is there a https site we could download these?

Comment (halkyon) [reply Dockerfile:62]: I don't think there is, maybe verification is done with checksum?

Comment (egonelbre) [reply Dockerfile:62]: Verifying the checksum would be fine as well, however I don't see it being verified here.

Comment (halkyon) [reply Dockerfile:62]: true, I'll get another change opened to add the checksum verification

Comment (halkyon) [reply Dockerfile:62]: https://github.com/storj/docker-golang/pull/21

**Pull request merged**